config/config.go

```diff
@@ -656,6 +657,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("external_url", "http://localhost:8000")
 	v.SetDefault("host", "")
 	v.SetDefault("port", 8000)
+	v.SetDefault("socket", "prebid-server.sock") // path of the socket's file which must be listened.
```

Comment:
I'd also like to see an option for enabling/disabling this functionality, defaulting to disabled.

Reply:
done, commit reference : [d26af7c]

enabling & disabling set into config_file and default_value "false".

Reply:
```Interesting. Do you have a need to listen both for http and unix sockets?```
I do not commit : [6838230] 
Handle this, if unix_socket enable, the main_server will be start as unix_socket and won't be start in TCP anymore.

```I understand the lack of unit tests for the listener if it's too hard to simulate, but can you please confirm if you've manually tested this feature end to end?```
Sure.